data/campaigns/Sceptre_of_Fire/scenarios/4_Gathering_Materials.cfg

```diff
@@ -233,7 +240,7 @@
         recruit=Troll Whelp,Young Ogre,Troll Rocklobber
         team_name=trolls1
         user_team_name=_"Resident Trolls"
-        {GOLD 75 125 175}
+        {GOLD 60 120 180}
```

Comment:
Could this make the later levels harder instead of easier? On easy, this looks like at least 36 xp less available to the player (one less troll whelp per team, multiplied by three teams).

Reply:
There are more turns on which the player can grind them though